examples/postgresql/README.md

```diff
@@ -23,7 +23,7 @@ $ helm repo add bitnami https://charts.bitnami.com/bitnami
 $ helm repo update
 
 $ kubectl create ns postgres-test
-$ helm install my-release --namespace postgres-test bitnami/postgresql
+$ helm install my-release --namespace postgres-test bitnami/postgresql --version 11.9.13
```

Comment:
Shouldn't we support the default version helm picks?

Reply:
fixed. updated dockerfile